git/helpers.sh

```diff
@@ -0,0 +1,59 @@
+#
+# Deletes a branch locally and remotely
+#
+function gitdelete() {
+  if [ -n "$1" ]; then
+    local_exists=`git show-ref refs/heads/$1`
+    remote_exists=`git ls-remote origin $1`
+    if [ -n "$local_exists" ]; then
+      git branch -D $1;
+    else
+      echo "could not find local branch: $1";
+    fi
+    if [ -n "$remote_exists" ]; then
+      git ps origin :$1;
+    else
+      echo "could not find remote branch: $1";
+    fi
+  fi
```

Comment:
might as well make local_exists and remote_exists uppercase as well.

Reply:
maybe we can also do `git remote prune origin`

Reply:
1- yes, for consistency (FYI, this method was not written by me ;-) )
2- I think pruning should be outside the scope of this function as it can lead to undesired outcome for the dev

Reply:
Okay. Yeah, that command would remove references to just the branch(es) they are working on.